### ISSUE TITLE
Invalid condenser outlet node conditions in 5ZoneBranchSupplyPumps.idf

### ISSUE THREAD

**Issue by jmarrec** [NatLabRockies/EnergyPlus#7134]
Issue overview
--------------

```
energyplus -w USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw 5ZoneBranchSupplyPumps.idf
```

With the file as is, no problem, eplusout.err is as clean as it can be.

Now just add this to the file:

```
  Output:Variable,*,System Node Relative Humidity, hourly;
```

Suddenly you get warnings:

```
   ** Warning ** Temperature out of range [-100. to 200.] (PsyPsatFnTemp)
   **   ~~~   **  Routine=NodeReportingCalc:CENTRAL CHILLER 1 CONDENSER OUTLET NODE,
   **   ~~~   **  Environment=RUNPERIOD 2, at Simulation time=07/07 07:00 - 07:05
   **   ~~~   **  Input Temperature=22699.39
   ************* Simulation Error Summary *************
   *************
   ************* ===== Recurring Error Summary =====
   ************* The following recurring error messages occurred.
   *************
   *************  ** Warning ** Temperature out of range [-100. to 200.] (PsyPsatFnTemp)
   *************  **   ~~~   **   This error occurred 84 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=79178.921875 C  Min=21399.591797 C
```

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@jmarrec Some node state variables are not calculated unless they are requested as output or used in EMS. This is done to save processing time for RH, wet-bulb temperature, dewpoint, and specific heat. This is managed in [`CalcMoreNodeInfo`](https://github.com/NREL/EnergyPlus/blob/5ddf585d0378775ebe219927560b332b6ed68d82/src/EnergyPlus/NodeInputManager.cc#L1213). 

That said, the error reveals that the conditions on the condenser outlet node are nonsense. I've changed the title of this to "Invalid condenser outlet node conditions in 5ZoneBranchPumps".

**Comment by jmarrec** [comment]
Yeah  I had started investigating this but haven't figured yet why the condenser node is actually 22000C.
